### PR TITLE
Master

### PR DESCRIPTION
**Pull request by LuckyCandice**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [.] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/analogix/analogix.quirk [hunk 1]
```diff
@@ -1,3 +1,15 @@
 # Phoenix-Lite
 [USB\VID_1F29&PID_7518]
 Plugin = analogix
+
+//Belkin
```
Comment (hughsie) [inline plugins/analogix/analogix.quirk:5]: can you use `#` for comments please

File: plugins/analogix/fu-analogix-device.c [hunk 2]
```diff
@@ -241,7 +241,24 @@ fu_analogix_device_probe (FuDevice *device, GError **error)
...
+	}
+	else if (vid == 0x047D)
+	{
+		fu_device_set_vendor (device, "Kensington");
+	}
+	else if (vid == 0x0502)
+	{
+		fu_device_set_vendor (device, "Acer");
+	}
+	else{}
```
Comment (hughsie) [inline plugins/analogix/fu-analogix-device.c:261]: Do you need an of these? The vendor should be bieng set automatically using the udev vendor database.

Comment (LuckyCandice) [reply plugins/analogix/fu-analogix-device.c:261]: Do you mean I don't need call fu_device_set_verdor to set vendor name? But if I didn't call this API, I found verdor is displayed as "Analogix".

Comment (hughsie) [reply plugins/analogix/fu-analogix-device.c:261]: Try removing the manual:

    > plugins/analogix/fu-analogix-device.c:  fu_device_set_vendor(FU_DEVICE(self), "Analogix Semiconductor Inc.");

**Comment by hughsie** [comment]
I don't think you wanted that last commit

File: plugins/analogix/fu-analogix-device.c (hunk 2 above)
Comment (LuckyCandice) [reply plugins/analogix/fu-analogix-device.c:260]: I removed this line codes, but the vendor displays "Analogix".

Comment (hughsie) [reply plugins/analogix/fu-analogix-device.c:260]: What does the USB descriptor say? i.e. the string pointed by iManufacturer?

Comment (LuckyCandice) [reply plugins/analogix/fu-analogix-device.c:260]: I see, let me check this, I will pull a new request, thank you!

**Pull request closed without merging**